src/app/features/courses-page/components/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './course-search/course-search.component';
```

Comment:
export public entities explicitly `export {Component} from './component.name'`

Reply:
@SteveRusin Made all exports to explicitly.